core/server/src/test/java/alluxio/master/file/FileSystemMasterTest.java

```diff
@@ -725,6 +726,21 @@ public void createFileWithTtl() throws Exception {
   }
 
   /**
+   * Tests that an exception is in the
+   * {@link FileSystemMaster#createDirectory(AlluxioURI, CreateDirectoryOptions)} with a TTL
+   * set in the {@link CreateDirectoryOptions} after the TTL check was done once.
+   */
+  @Test
+  public void createDirectoryWithTtl() throws Exception {
+    CreateDirectoryOptions directoryOptions =
+            CreateDirectoryOptions.defaults().setRecursive(true).setTtl(0);
```

Comment:
this is not the correct indentation (Alluxio Java style guideline is to use 4 spaces), please fix this through all the newly contributed code